### DISCUSSION TITLE
Calendar view of reminders

### DISCUSSION THREAD

**Discussion by pmay68** [hargata/lubelog#280] [Ideas]
I hope I am placing this in the right location.   We here at Southwest Metal have been using lube logger to keep track of our vehicles, trailers, forklifts etc.

Everyone uses this to know when something is coming due so we can plan around it if a piece of equipment needs to go down for maintenance.  Having a calendar view of all the reminders would give our people a birds eye view of the month and what needs to be done with our equipment ahead of time in order to plan accordingly.

The reminders are great, but they only help to some extent.   Being able to look at a calendar and see all reminders for a specific month would be a great asset and help to all of us here.

I would be happy to contribute to this addon if we could get this included.

**Comment by hargata** [comment]
@pmay68 
This feature will be released later tonight, it's going to be at the garage level, and this is very important, when this thing is released and you updated your copy of LubeLogger, you and anyone who is going to be accessing your instance of lubelogger are going to have to clear your browser cache. Meaning to clear cookies, etc, otherwise the app is going to be acting weird on their end. Sometimes on some devices, the browser will persist despite of that, one way to make sure the issue on the user's end is to have them access LubeLogger in Incognito mode.

![image](https://github.com/hargata/lubelog/assets/155338622/3560d417-571a-4d32-8d48-251323ffe889)

**Reply by pmay68** [reply]
This is awesome and is going to be a huge help here for our team.  Is there a way to wrap the txt so that is stays in in the cell without expanding it? maybe a continue button if there is not enough space in the cell rather than the cell expanding the length of the information?   In December we have like 20 reminders from registrations to inspections etc. so this literally takes up the entire side of a page listing all the reminders.   Was thinking a continue button to show in a pop up maybe what's unable to fit in that cell.

Again, thanks for this upgrade, where can I contribute.  I would like to send you some funds for doing this for us.

**Comment by pmay68** [comment]
Oh and maybe when you click on the item in calendar it takes you to that reminder so you can edit or push to another date?

**Reply by hargata** [reply]
Hey, so along with your suggestions above, this is what it will look like when I release this later tonight:

![image](https://github.com/hargata/lubelog/assets/155338622/826140e9-282c-484e-9ef2-e81d6df80636)

So you can kinda see that when it overflows, it adds a scrollbar, and when you click on a reminder, it shows a very simplified, read only view that will give you more info regarding the reminder:

![image](https://github.com/hargata/lubelog/assets/155338622/1661e385-2c99-4913-8e86-0ce5098f23b3)

You can delete the reminder, or if the reminder is recurring, a button will be available where you can push it back by marking it as done.

As for monetary support, commercial / corporate users are required to subscribe to [our Patreon](https://www.patreon.com/LubeLogger) for $2.99/month

If you wish to make a donation, we also accept donations on [Stripe](https://buy.stripe.com/aEU9Egc8DdMc9bO144)

**Reply by pmay68** [reply]
Tha is great, I sent you 50.00 for now and will be subscribing to the monthly.   Thank you!!

**Reply by hargata** [reply]
Thank you!
